app/pages/returns/NonExportedHumanMedicinesPlasticPackagingWeightPage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.returns
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object NonExportedHumanMedicinesPlasticPackagingWeightPage extends QuestionPage[Long] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "nonExportedHumanMedicinesPlasticPackagingWeight"
+}
```

Comment:
Do we need a cleanup here if a user decides to change link to nonExportedHumanMedicinesPlasticPackagingWeight?

Reply:
Change and clean up is up for discussion at the moment s owe will come back to this.